docs/source/release_notes/v0.2.0.md

```diff
@@ -0,0 +1,33 @@
+
+Please visit [GETTING STARTED](/getting_started/index) guide and follow tutorials.
+
+You can learn more about SDK usage through [DEVELOPING WITH SDK](/developing_with_sdk/index).
+
+Please let us know how you like it and what could be improved by [submitting an issue](https://github.com/Project-MONAI/monai-deploy-app-sdk/issues/new/choose) or [asking questions](https://github.com/Project-MONAI/monai-deploy-app-sdk/discussions)😀
+
+## What's new in this version 0.2.0
+### Series Selection Operator
+This is to support the use case where whole DICOM studies are used as input to an AI inference application even though only specific series are applicable.
+
+The selection rules are defined in JSON, allowing multiple selections, each with a set of matching conditions. The rules processing engine is implemented in the `DICOMSeriesSelectorOperator`, which itself is regarded as a base class with default implementation. More advanced rules and processing engine can be implemented in derived classes.
+
+Multiple instances of the series selection operators, each having its own rules, can be chained in a MONAI Deploy application. In part this is made possible by the new App SDK Domain classes which encapsulate the selected series in a DICOM study, and are used as the output of each series selection operator.
+
+### DICOM Comprehensive Structured Report Writer
+This is introduced to support generating DICOM SR SOP instances for AI classification result, and as such, the DICOM SR writer is limited to support textual results.
+
+The DICOM SR writer is implemented in `DICOMTextSRWriterOperator`, it
+- loads AI result from in-memory object as well as from file path, with memory taking precedence
+- copies applicable DICOM tags from the original DICOM series used as input for the inference application, as well as generating tags anew when there is no DICOM series provided.
+- supports assigning DICOM tags via a dictionary with DICOM keywords and value, so that an application can customize the tags in the DICOM SR instance
+- provides classes for an application to encapsulate the AI model information as well as DICOM equipment information, per [IHE Radiology Technical Framework Supplement AI Results (AIR)](https://www.ihe.net/uploadedFiles/Documents/Radiology/IHE_RAD_Suppl_AIR.pdf)
+
+### Updated example applications
+- [The AI Spleen Segmentation](https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/main/examples/apps/ai_spleen_seg_app) application updated to demonstrate the use of series selection rules
+- [The MedNIST Classifier application](https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/main/examples/apps/mednist_classifier_monaideploy) updated to demonstrate the use of DCIOM SR writing (without initial DICOM input)
+- Updated are the main functions of the built-in [operators](https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/main/monai/deploy/operators), which serve as examples on parsing the output objects
```

Comment:
```suggestion
- Updated are the main functions of the built-in [operators](https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/main/monai/deploy/operators), which serve as examples of parsing the output objects
```

Reply:
changed to `examples on how to parse`, wanted to emphasize the `how`, as the Domain classes are getting more complex now, mingled with native classes from underlying packages.